src/complex/Core/Application.cpp

```diff
@@ -111,6 +112,45 @@ std::filesystem::path Application::getCurrentDir() const
   return m_CurrentPath.parent_path();
 }
 
+std::optional<Uuid> Application::getComplexUuid(const Uuid& simplUuid)
+{
+  std::string searchUuid = simplUuid.str();
+  if(m_SIMPL_To_Complex.count(searchUuid) == 1)
```

Comment:
For early returns, I think that using the error condition is better. Code with less nesting is easier to read.
```cpp
if(m_SIMPL_To_Complex.count(searchUuid) == 0)
{
  return {};
}
return Uuid::FromString(m_SIMPL_To_Complex[searchUuid]);
```
The same kind of change can be made in the other function.